internal/host/transition_test.go

```diff
@@ -3797,7 +3797,40 @@ var _ = Describe("Refresh Host", func() {
 			},
 		}
 
-		for _, hostName := range []string{"localhost", "localhost.localdomain", "localhost4", "localhost4.localdomain4", "localhost6", "localhost6.localdomain6", "AAAA.com", "redhat.com."} {
+		for _, hostName := range []string{"localhost", "localhost.localdomain", "localhost4", "localhost4.localdomain4", "localhost6", "localhost6.localdomain6"} {
```

Comment:
We can reuse the existing hostnames loop and change it so it loops over anonymous {hostname string, is_forbidden bool} structs, it can avoid a lot of duplication